torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -9,7 +9,7 @@
 #include "pjrt_computation_client.h"
 #include "torch_xla/csrc/runtime/computation_client.h"
 #include "torch_xla/csrc/runtime/debug_macros.h"
-#include "torch_xla/csrc/runtime/distributed_runtime.h"
+//#include "torch_xla/csrc/runtime/xla_coordinator.h"
```

Comment:
Can you uncomment this?

Reply:
Oops, I should have proofread before requesting review, thanks Will!